development-updates.md

```diff
@@ -14,6 +14,10 @@
   - [Anthony Madia](#anthony-madia)
     - [Update #1](#anthony-madia-update) - [Update #2](#anthony-madia-update-1)
   - [Bem Baraki](#bem-baraki)
+    - [Update #1](#anthony-madia-update)
```

Comment:
Please point this to the right tag with Bem's updates

Reply:
I've rebased and Bem's updates are now pointed to the right tag